skll/config.py

```diff
@@ -33,6 +35,19 @@
                                             'none'])
 
 
+def _transform_learner_name(learner_name):
```

Comment:
I think you should just snatch the class out of `globals()` rather than making your own dict and using this transformer thing.